### PR TITLE
Add separate status change action and option to not run immediately on moving to processing

### PR DESCRIPTION
**Pull request by Mark-H**
@isaacniebeling

### CODE REVIEW

File: core/components/commerce_usergroupshipment/model/commerce_usergroupshipment/usergroupshipment.class.php [hunk 1]
```diff
@@ -31,7 +33,14 @@ public static function getFieldsForProduct(Commerce $commerce, comProduct $produ
 
     public static function getFieldsForDeliveryType(Commerce $commerce, comDeliveryType $deliveryType)
     {
-        return [];
+        $fields = [];
+        $fields[] = new CheckboxField($commerce, [
+            'name' => 'properties[run_automatically]',
```
Comment (isaacniebeling) [inline core/components/commerce_usergroupshipment/model/commerce_usergroupshipment/usergroupshipment.class.php]: this should be `run_ugs_automatically`

**Comment by isaacniebeling** [comment]
image

lexicon entry is missing (again something i could potentially update, but i just don't want to lose track of it)

**Comment by isaacniebeling** [comment]
image

settings not using lexicons

**Comment by isaacniebeling** [comment]
@Mark-H If I get those updates in, any thoughts on when this could be packaged and released? I did some testing when I commented here and it looks like it worked then? 😂 [Slack link to relevant time](https://modmore.slack.com/archives/C1LJ5Q8HF/p1618587055131400)

**Comment by Mark-H** [comment]
Releasing takes like 2 minutes. It's the testing, updating docs, etc that takes most time. ;)

**Comment by isaacniebeling** [comment]
> settings not using lexicons

Since the settings are loaded from a static call in comProduct->getModelFields, I'm not sure how to load things from the adapter's lexicon...

**Comment by Mark-H** [comment]
`$commerce->adapter`

**Comment by isaacniebeling** [comment]
> `$commerce->adapter`

doh. thanks.

**Comment by isaacniebeling** [comment]
In my testing, this appears to work. I've added to the documentation as a PR as well. Sooooooooooooooooo 😂

**Pull request merged**